states/persistenceDiagramDistance.pvsm

```diff
@@ -86387,7 +86387,7 @@
     </Proxy>
     <Proxy group="filters" type="ttkPersistenceDiagram" id="21305" servers="1">
       <Property name="BackEnd" id="21305.BackEnd" number_of_elements="1">
-        <Element index="0" value="0"/>
+        <Element index="0" value="2"/>
```

Comment:
for this example, the diagrams needs to be computed from the opposite of the function.

Reply:
Seems a bit uglier to me... Maybe also show the saddle-saddle pairs? There's one pair in particular that has a high persistence in both datasets.

Reply:
agreed. could you add the saddle-saddle pairs really quick?